bin/generateChromeManifest.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+
+##
+# This script is designed to generate the chrome.manifest for the add-on in
+# the propert format, taking into account the locales present in the repo.
+##
+
+from __future__ import print_function
+
+import argparse
+import io
+import os
+import re
+
+# defaults
+DEF_L10N_SRC = os.path.join("locale")
+DEF_INPUT_FILE_NAME = os.path.join("add-on", os.extsep.join(["jar", "mn"]))
+DEF_OUTPUT_FILE_NAME = os.path.join("built", "add-on", os.extsep.join(["chrome", "manifest"]))
+
+
+def main(l10n_src, input_file_name, output_file_name):
+    print("Using l10n tree from", l10n_src)
+
+    locale_dirs = sorted(os.listdir(l10n_src))
+
+    input_file = io.open(input_file_name, "r")
+
+    jar_mn = input_file.read()
+
+    # sed -n -e '/^%/p' $(ADD-ON)/jar.mn | \
+    # sed -e "s/^% //g" | sed -e "s/%/chrome\//g" \
+
```

Comment:
These `sed` comments probably aren't as useful as the expanded comments below. ;)